src/common/internalLink/internalLink.module.scss

```diff
@@ -5,6 +5,13 @@
 
   text-decoration: none;
   color: var(--vene-internal-link-color);
+
+  display: flex;
+  align-items: center;
+
+  & > :first-child {
+    margin-right: 1rem;
```

Comment:
Can you use the spacing units we're using in other places to keep the interface consistent?

Reply:
using spacing units here